tasks.py

```diff
@@ -53,3 +75,55 @@ def integrations_changed(ctx):
         if line:
             checks.add(line.split('/')[0])
     return checks
+
+
+@task(help={
+    'package': 'The package to upgrade throughout the integrations',
+    'version': 'The version of the package to pin',
+    'verbose': 'Whether or not to produce output',
+})
+def upgrade(ctx, package, version, verbose=False):
+    here = os.path.dirname(os.path.abspath(__file__))
+
+    for check_name in sorted(os.listdir(here)):
+        check_dir = os.path.join(here, check_name)
+        reqs_in = os.path.join(check_dir, 'requirements.in')
+        reqs_txt = os.path.join(check_dir, 'requirements.txt')
+
+        ensure_deps_declared(reqs_txt, reqs_in)
+
+        if os.path.isfile(reqs_in):
+            with open(reqs_in, 'r', encoding='utf-8') as f:
+                lines = f.readlines()
+
+            for i, line in enumerate(lines):
+                try:
+                    pkg = line.split('=')[0].strip()
+                    if pkg == package:
+                        break
+                except IndexError:
+                    continue
+            else:
```

Comment:
Can you put a comment here about this else clause? I had to look it up to be sure that I was remembering correctly what for/else did in python and I can see this easily being accidentally removed.